### ISSUE TITLE
Top Nav - 1.4.10 Reflow - Login button text is cut off at 300%. All content should be visible and operable at up to 400% zoom.

### ISSUE THREAD

**Issue by johnatawnclementawn** [archesproject/arches#9839]
In both logged in and logged out states, the button's text is not replaced by an icon at smaller screen sizes

![Image](https://github.com/archesproject/arches/assets/22161487/dfc140e3-ff1e-4eae-bb63-e7e70d795699)

**Comment by phudson-he** [comment]
@johnatawnclementawn In a logged in state, The "Welcome, User" text is displayed and is just a link to the users profile page. The thought process around this for smaller width devices was that there is already a link to the users profile in the left nav, so the welcome message isn't really required any more, so was hidden as it takes up valuable screen real estate. What are your thoughts on this? - Perhaps the hidden-xs should be on the containing element so that the empty box in not rendered.

With regards to the logged out state, I agree that a suitable login icon should replace the "Login" text, so I will work on that.

**Comment by johnatawnclementawn** [comment]
@phudson-he Totally agree - I like the idea of applying hidden-xs to containing element. This will likely necessitate applying that css dynamically based on a user's logged-in state, in order to show the 'Login' icon

**Comment by johnatawnclementawn** [comment]
Resolved in #10291
